packages/react/src/context/test-slash-id-provider.tsx

```diff
@@ -10,11 +10,46 @@ type TestProviderProps = Partial<ISlashIDContext> & {
   children: React.ReactNode;
 };
 
-export const TEST_TOKEN =
-  "eyJhbGciOiJSUzI1NiIsICJraWQiOiJ2RXBzRmcifQ.eyJhdXRoZW50aWNhdGVkX21ldGhvZHMiOlsiZW1haWxfbGluayJdLCAiZXhwIjoxNjY4Nzc1NjE3LCAiZmlyc3RfdG9rZW4iOmZhbHNlLCAiZ3JvdXBzIjpbXSwgImlhdCI6MTY2ODc2NzAxNywgImlzcyI6Imh0dHBzOi8vc2FuZGJveC5zbGFzaGlkLmRldiIsICJqdGkiOiIwMzliYmIyN2MwN2I2OWIyMTc2MmVjMGExYTljMmRjNSIsICJvaWQiOiJhZDUzOTllYS00ZTg4LWIwNGEtMTZjYS04Mjk1OGM5NTU3NDAiLCAidXNlcl9pZCI6InBpZDowMWU0M2YyNDE5ZmU5OTQ4NzlhNjQ1NjRjZDc2YWIzMGE4ZDJlYTk1ZTg5OTg3YzgxODVjZWY1YWI4ZjhhZGY4ZGU2NDNhMjg5YzoyIn0.tsyUk3guY29r-jb-Xw2htT0egEO3KUErDSlJu9F9Y_QQAf6Te_DmdPgnCKjR7pTGO1uKvYT6JKit7opyntOA4y_wIhymUOkW5mtX-fgyIF0Fkxx1JjGm4BcTE9rI1tH7DWG177yTzwJ2kv5OYvTknpn_QK8s6JzD1N5Yq11_VNf2dRN_NXb-0feqDGhXU7lR-oO7wqFlt37pzENQ7-tG3JDt9uCKqSbrtXqxTHGtg80ZY3FxXYYiHNC3v0nXV5aFRhxGvIIm9LgNkZwXkEtSecIqFHWJn2-ILuOFpvcmtmlZr8AxQyNMAKMt1fARf2LJy45qITI2IyVTndtDekT6HQ";
+const TEST_TOKEN_HEADER = "eyJhbGciOiJSUzI1NiIsICJraWQiOiJ2RXBzRmcifQ";
+const TEST_TOKEN_PAYLOAD_DECODED = {
+  authenticated_methods: ["email_link"],
+  exp: 1668775617,
+  first_token: false,
+  groups: [],
+  iat: 1668767017,
+  iss: "https://sandbox.slashid.dev",
+  jti: "039bbb27c07b69b21762ec0a1a9c2dc5",
+  oid: "ad5399ea-4e88-b04a-16ca-82958c955740",
+  user_id:
+    "pid:01e43f2419fe994879a64564cd76ab30a8d2ea95e89987c8185cef5ab8f8adf8de643a289c:2",
+};
+
+const TEST_TOKEN_SIGNATURE =
+  "tsyUk3guY29r-jb-Xw2htT0egEO3KUErDSlJu9F9Y_QQAf6Te_DmdPgnCKjR7pTGO1uKvYT6JKit7opyntOA4y_wIhymUOkW5mtX-fgyIF0Fkxx1JjGm4BcTE9rI1tH7DWG177yTzwJ2kv5OYvTknpn_QK8s6JzD1N5Yq11_VNf2dRN_NXb-0feqDGhXU7lR-oO7wqFlt37pzENQ7-tG3JDt9uCKqSbrtXqxTHGtg80ZY3FxXYYiHNC3v0nXV5aFRhxGvIIm9LgNkZwXkEtSecIqFHWJn2-ILuOFpvcmtmlZr8AxQyNMAKMt1fARf2LJy45qITI2IyVTndtDekT6HQ";
+
+export const TEST_TOKEN = [
+  TEST_TOKEN_HEADER,
+  btoa(JSON.stringify(TEST_TOKEN_PAYLOAD_DECODED)),
+  TEST_TOKEN_SIGNATURE,
+].join(".");
 
 export const TEST_USER = new User(TEST_TOKEN);
 
+export function NewTestUserWithAuthMethods(authMethods: FactorMethod[]): User {
```

Comment:
This is a factory function so by convention it should be named with `create*` prefix. For example:
```
export function createTestUser({authMethods: FactorMethod[]}): User // ...
```

Reply:
Also it would be useful to document that this will not generate a valid token, just one that can be decoded and used in the SDK for testing.

Reply:
I quite like the idea btw 👏